packages/ember-metal/lib/computed.js

```diff
@@ -23,7 +23,7 @@ import {
 */
 
 
-function UNDEFINED() { }
+export const UNDEFINED = symbol('undefined');
```

Comment:
should i used the shared existing UNDEFINED from https://github.com/emberjs/ember.js/blob/27d67841783004538ce0096083b37c6d4e96e824/packages/ember-metal/lib/meta.js#L282 ?